src/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Provider} from 'react-redux';
+
+import {App} from 'components/app';
+
+import {store} from './store/store-config';
+
+const rootNode = document.getElementById('root'); // eslint-disable-line no-undef
```

Comment:
You have to avoid inline unabling eslint rules. Try to find better solution and change config for example